### PR TITLE
DOC-1708: split out help and revised content as necessary.

### PR DESCRIPTION
**Pull request by tvo318**
@dkavanagh: Can you review and merge please? Thank you!!!

### CODE REVIEW

**Comment by euca-qa-gui-tester** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.qa1.eucalyptus-systems.com/job/user-console-PR-monitor/1353/
Test PASSed.

File: eucaconsole/static/html/help/console_create_instances.html [hunk 1]
```diff
@@ -0,0 +1,294 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (dkavanagh) [inline eucaconsole/static/html/help/console_create_instances.html]: This file seems to be all of the separate tabs instructions in one file, right? Weren't we splitting, or is this not for the launch instance wizard?

File: eucaconsole/static/html/help/console_create_launch_config.html [hunk 2]
```diff
@@ -0,0 +1,222 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (dkavanagh) [inline eucaconsole/static/html/help/console_create_launch_config.html]: same comment as above. this looks like a unified file when we're trying to split.

File: eucaconsole/static/html/help/console_create_launch_config_from_instance.html [hunk 3]
```diff
@@ -0,0 +1,188 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (dkavanagh) [inline eucaconsole/static/html/help/console_create_launch_config_from_instance.html]: same comment as above. this looks like a unified file when we're trying to split.

**Comment by dkavanagh** [comment]
It looks like the only new files are ones that duplicate material from split files already in use. I'd remove those files (I commented on them) and I think we're good to go.

File: eucaconsole/static/html/help/console_create_instances.html (hunk 1 above)
Comment (tvo318) [inline eucaconsole/static/html/help/console_create_instances.html]: This file should no longer be used, according to the ticket, it was split out to:
Image (console_create_instances_image.html)
Details (console_create_instances_details.html)
Security (console_create_instances_security.html)
Advanced (console_create_instances_advanced.html)

File: eucaconsole/static/html/help/console_create_launch_config.html (hunk 2 above)
Comment (tvo318) [inline eucaconsole/static/html/help/console_create_launch_config.html]: This file is no longer used, according to the doc ticket, it was split out to:
Image (console_create_launch_config_image.html)
Details (console_create_launch_config_details.html)
Security (console_create_launch_config_security.html)
Advanced (console_create_launch_config_advanced.html)

File: eucaconsole/static/html/help/console_create_launch_config_from_instance.html (hunk 3 above)
Comment (tvo318) [inline eucaconsole/static/html/help/console_create_launch_config_from_instance.html]: This file is no longer used, this wizard now uses the create launch configuration help:
Image (console_create_launch_config_image.html)
Details (console_create_launch_config_details.html)
Security (console_create_launch_config_security.html)
Advanced (console_create_launch_config_advanced.html)

**Comment by tvo318** [comment]
The files that should NOT be used are:

- console_create_instances.html
- console_create_launch_config
- console_create_stack

All the details tabs are split out as well. Please see DOC-1708.

File: eucaconsole/static/html/help/console_create_instances.html (hunk 1 above)
Comment (dkavanagh) [inline eucaconsole/static/html/help/console_create_instances.html]: Right, so I was curious why it was being added in this PR?

**Comment by euca-qa-gui-tester** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.qa1.eucalyptus-systems.com//job/user-console-PR-monitor/1548/
Test FAILed.

**Pull request merged**